app/models/user.rb

```diff
@@ -78,30 +78,6 @@ def all_hooks
     all
   end
 
-  def payload
-    attrs = { "id" => id, "handle" => handle }
-    attrs["email"] = email unless hide_email
-    attrs
-  end
-
-  def as_json(*)
-    payload
-  end
-
-  def to_xml(options = {})
-    payload.to_xml(options.merge(root: 'user'))
-  end
-
-  def to_yaml(*args)
-    payload.to_yaml(*args)
-  end
```

Comment:
> It just doesn't seem an obvious improvement here

I am hoping that we can avoid overriding `encode_with`. It gets in the way of serialization for delayed_job. For example, we want to be able to call `user.delay.destroy`. Right now, it fails because it doesn't know how to call `destroy` on hash `{id: .., handle: .., email: ..}`.
@indirect argued that
> background jobs do not serialize the object at all. Instead, pass the object ID, and use ActiveRecord to find a new copy of the object inside the background job before doing operations on it. 

However, it is really not what I am observing.